Autopilot/boardfiles/Src/freertos.c

```diff
@@ -50,12 +50,18 @@
 
 /* Includes ------------------------------------------------------------------*/
 #include "FreeRTOS.h"
+#include "portmacro.h"
 #include "task.h"
 #include "main.h"
 #include "cmsis_os.h"
 
 /* Private includes ----------------------------------------------------------*/
 /* USER CODE BEGIN Includes */
+#include "attitudeManagerInterface.hpp"
+#include "PathManagerInterface.h"
+#include "telemetryManagerInterface.h"
+#include "sensorFusionInterface.hpp"
+#include "SensorFusion.hpp"
```

Comment:
I think this is you issue actually. Your'e including SensorFusion.hpp here. You shouldn't have to. you only need the interface.